index.html

```diff
@@ -42,10 +42,27 @@ <h1>
         </ul>
       </nav>
     </div>
+    <section class="flex-container">
+      <div id="container">
+    <!-- Each image is 300px by 200px -->
+        <div class="photobanner">
+          <img class="first" src="/images/wedding reception invitation card potential photos/all/IMG_1379.JPG" height="200" alt="cheese"/> 
```

Comment:
@anitasunshine - To make development easier, for now I think we should drop the leading `/` from the `src`. The leading slash indicates that the path should be relative to the root, but the root will different depending on whether you're working on your machine or if it's the actual site.

I wasn't able to load the images locally because `/` resolves to the root directory of my machine, which doesn't have an `images` directory. However, this isn't a problem on the actual site, since it'll resolve relative to the main page.

@aakside - what do you think?

Reply:
Agreed. Make sure paths don't begin with `/`

Reply:
Confirmed that this was resolved.